Vertexvis/vertex-web-sdk
# [PLAT-1586] Support zooming past hit points

## Summary

Adds a `useMinimumPerspectiveZoomDistance` flag (defaulted to true) to allow a zoom interaction to zoom past the determined hit point. This allows for a continual zooming experience when the initial interaction was started with the cursor over geometry closer to the camera than the ultimate destination. When `useMinimumPerspectiveZoomDistance` is false, zoom will behave as it did previously, but with the flickering addressed when too close to the hit point.

This PR also attempts to fix one more scene-tree spinner 🙂 

## Test Plan

- Verify that zooming now allows you to pass through objects/hit planes
- Verify that setting `useMinimumPerspectiveZoomDistance` to false causes zoom to behave as it did previously

## Release Notes

N/A

## Possible Regressions

Zoom interactions

## Dependencies

N/A
